src/api/routes/ws/socket-io.ts

```diff
@@ -64,6 +95,23 @@ export function createSocketIORouter(db: DataStore, server: http.Server) {
     });
   });
 
+  io.use((socket, next) => {
```

Comment:
Add a small comment above this line to explain what you'll be doing with this middleware.